src/for-developers/sway.md

```diff
@@ -0,0 +1,11 @@
+# Sway Language 🌴
+
+Resources to get started with Sway:
+
+- [Official Sway docs](https://fuellabs.github.io/sway/v0.19.2/)
+- [Sway examples repo](https://github.com/FuelLabs/sway/tree/376010e7b8f84334ffacd0f99e64f185c971d213/examples)
```

Comment:
In this case, there's very little chance we'll rename the examples directory, so a permalink isn't necessary.

```suggestion
- [Sway examples repo](https://github.com/FuelLabs/sway/tree/master/examples)
```